js_basics_HT/task5_2.js

```diff
@@ -0,0 +1,5 @@
+const lenght = 2
+const weight = 3
+const SR = (lenght*weight)
```

Comment:
```js
const reactangleArea = lenght*weight
```

Reply:
remember about camelCase and meaningful names for variables

Reply:
*rectangle
SR - means rectangle square, I'll replace it with rectangleArea and weight with width